components/mpas-albany-landice/src/Registry.xml

```diff
@@ -1471,6 +1477,27 @@ is the value of that variable from the *previous* time level!
 	       />
         </var_struct>
 
+<!-- ================ -->
+
+        <!-- Variables related to time averaging of fluxes -->
+        <var_struct name="timeAveraging" time_levs="1">
+                <var name="timeAccumulatedCoupled" type="real" dimensions="Time" units="s"
+                     description="accumulated time for use in averaging of coupler fields"
+                />
+                <var name="avgCalvingThickness" type="real" dimensions="nCells Time" units="kg m^{-2} s^{-1}"
+                     description="time averaged thickness of ice that calves (using calvingThickness from 'geometry' pool)"
+                />
+                <var name="avgFaceMeltingThickness" type="real" dimensions="nCells Time" units="m" time_levs="1"
```

Comment:
Incorrect units, I think.

Reply:
True. These are the units after we multiply the thickness value by "kg/m^3 / s". What we do here depends a bit on whether we decide to rename these vars. or not (as you note above). Another option would be to keep them (because they are at least correct in their description for what we are actually calculating here -- i.e., a loss by calving if it were applied as a thickness value over an entire grid cell) and add new fields that are simply these fields converted to the correct units for a mass flux.